.eslintrc

```diff
@@ -2,7 +2,8 @@
   "rules":{
     "no-console": 0,
     // Inherited from airbnb via eslint-config-amo
-    "function-paren-newline": 0
+    "function-paren-newline": 0,
+    "import/no-dynamic-require": 0
```

Comment:
Let's maybe not generalize this but make use of a `// eslint-disable-next-line import/no-dynamic-require` comment right before the line you're going to use that dynamic import.

You're already doing that so adding the ignore here shouldn't have any impact.

Reply:
Ya I did commit that. Forgot to push I think